sphinxcontrib_images_lightbox2/lightbox2.py

```diff
@@ -23,7 +23,8 @@ def visit_image_node_html(self, writer, node):
                                        builder.images[node['uri']])
 
         writer.body.append(
-            u'''<a data-lightbox="{group}"
+            u'''<figure class="{cls}">
```

Comment:
I think we need to use ``cls`` only in one place, figure or a, not both, what do you think? It can break styling?